packages/queued-request-controller/src/QueuedRequestController.ts

```diff
@@ -29,9 +41,23 @@ export type QueuedRequestControllerEvents =
 export type QueuedRequestControllerActions =
   QueuedRequestControllerEnqueueRequestAction;
 
+type AllowedActions =
+  | NetworkControllerGetStateAction
+  | NetworkControllerSetActiveNetworkAction
+  | NetworkControllerSetProviderTypeAction
```

Comment:
this controller doesn't use any of these actions though.

Reply:
Is there a different name that you could suggest for this type union? I think we should avoid relying on copy-pastes to express this abstraction.

Reply:
@BelfordZ Would you mind taking a second look at this when you're free? 

Based on [`createQueuedRequestMiddleware`](https://github.com/MetaMask/core/pull/1890/files#diff-02c32f437b122b5c71f0ac6bdf304cd22f6d4569c9c9b8ecbb711c11c98cd0d3L51-L67) and [`buildMessenger`](https://github.com/MetaMask/core/pull/1890/files#diff-d85004a6e4bc8c032dffe450176f16ba4dad4806dce07bd2b2bb9838d73ca04aL23-L34), it looks like the messengers for `QueuedRequestController` need access to these `AllowedActions`, but I might be misunderstanding the context here.